anvil_consortium_manager/templates/anvil_consortium_manager/workspace_landing_page.html

```diff
@@ -0,0 +1,52 @@
+{% extends "anvil_consortium_manager/base.html" %}
+{% load static %}
+
+{% block title %}Workspaces{% endblock %}
+
+{% block content %}
+
+
+<h1 class="pb-2 border-bottom">Workspaces</h1>
+
+
+<div class="album py-2">
+  <div class="container">
+    <div class="row row-cols-1 row-cols-sm-2 row-cols-md-3 g-3">
+
+      {% for workspace_type, workspace_name in registered_workspaces.items %}
```

Comment:
Try passing this as a context variable in the view instead of using the context processor. Then you can test that that variable contains the correct set of workspace adapters/types.

That way we might be able to remove the context processor entirely.